R/bag_tree-rpart.R

```diff
@@ -50,15 +50,25 @@ get_missings_survbagg <- function(object, new_data) {
 #' A wrapper for survival probabilities with `survbagg` models
 #' @param object A model from `ipred::bagging()`.
 #' @param new_data Data for prediction.
-#' @param time A vector of prediction times.
+#' @param eval_time A vector of prediction times.
+#' @param time Deprecated. A vector of prediction times.
```

Comment:
In these `@param` entries, is it worth saying deprecated _in favor of `eval_time`_? The error message is clear here, so maybe this isn't needed.

Reply:
The warning message is clear but better to be so clear that nobody needs to trigger that warning in the first place - adding it 👍